### DISCUSSION TITLE
Quiet state toggle button?

### DISCUSSION THREAD

**Discussion by sqmagellan** [jeffschubert/homebridge-daikin-oneplus#43] [Ideas]
I've noticed the most aggressive version of quiet mode is very energy efficient for the Daikin Fit, but sometimes I need to turn quiet mode off to get some burst cooling/heating on demand.

Do you know if the Daikin API exposes quiet mode, and if so is it possible to make a quiet state toggle button?

**Comment by jeffschubert** [comment]
@sqmagellan while the API does expose the quiet mode parameters, I haven't figured out the correct properties/values to send in order to toggle it on/off. If I can figure it out, I've got the code ready to commit and add a switch for it.
